components/flow-orchestration-framework/org.wso2.carbon.identity.flow.execution.engine/src/main/java/org/wso2/carbon/identity/flow/execution/engine/FlowExecutionService.java

```diff
@@ -140,6 +140,13 @@ public FlowExecutionStep executeFlow(String tenantDomain, String applicationId,
 
     private boolean enterFlowInIdentityContext(String flowType) {
 
+        if (StringUtils.isBlank(flowType) ||
+                Arrays.stream(org.wso2.carbon.identity.flow.mgt.Constants.FlowTypes.values())
```

Comment:
why is this package qaulified?
is there another FlowTypes in the same class?

Reply:
Updated